pkg/infoschema/test/clustertablestest/cluster_tables_test.go

```diff
@@ -855,7 +863,13 @@ func TestMDLView(t *testing.T) {
 			tk3 := s.newTestKitWithRoot(t)
 			tk.MustExec("use test")
 			tk.MustExec("set global tidb_enable_metadata_lock=1")
-			tk.MustExec(c.createTable)
+			for _, cr := range c.createTable {
+				if strings.Contains(c.name, "err") {
+					_, _ = tk.Exec(cr)
```

Comment:
check it does return err?

Reply:
Not all the SQLs return an error, I don't think it's necessary to check the error since it is just a preparation.